### DISCUSSION TITLE
Pronounciation suggestion: 'pst' like in 'lapsed'

### DISCUSSION THREAD

**Discussion by shaunlebron** [typst/typst#4369] [General]
The pronounciation guide at https://typst.app/legal/brand/ says:

> 'pst' like in Hipster.

_As a native-english speaking American_, it’s hard to stop at the "T" when pronouncing hipster in English, but it clicked for me when I thought of "lapsed" since it’s the only english word that ends in /pst/:

> 'pst' like in 'lapsed'.

Collapsed or relapsed also work.

_Previous discussion: https://github.com/typst/typst/discussions/2598_

**Comment by sitandr** [comment]
I'm not sure, but I think that could create a wrong impression for people with a pretty bad English. I think it is quite common for newbies to read "ed" ending always with a little [I] sound. Additionally, that's a bit less obvious in general due to spelling differences.

Personally, I had no problems with "Hipster" example, but had to wrap my head around "lapsed" a bit.

**Reply by shaunlebron** [reply]
Great, so I think this means "hipster" works better for non-native english speakers, while "lapsed" is the only one that works for native-english speaking Americans.  However ya’ll want to do it.  Just wanted to let you know how Americans hear it.
